app/controllers/registrations_controller.rb

```diff
@@ -11,10 +11,12 @@ def new
     respond_with resource
   end
 
+  # MARK: only adopters are created through this route. Adopters need both the adoper role and a form submission to attach their applications to
   def create
     super do |resource|
       if resource.persisted?
         resource.add_role(:adopter, Current.organization)
+        FormSubmission.create!(person_id: resource.person_id)
```

Comment:
Can we call the associations here?
e.g. `resouce.person.create_form_submission`